src/data_processing/nihr_data.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+
+
+def read_nihr_dataset():
+    """Reads NIHR dataset and clean columns.
+
+    Returns:
+        pd.DataFrame: Cleaned NIHR dataset.
+
+    """
+    nihr_df = pd.read_parquet(
+        'data/raw/nihr_all.parquet',
```

Comment:
similarly, to the `ukhra_data.py` comment, I would make this into an argument, first of the definition but ultimately controllable from the Makefile via click or Argparse